release/release_logs/2.0.0/microbenchmark.json

```diff
@@ -1,274 +1,274 @@
 {
     "1_1_actor_calls_async": [
-        6146.424440513006,
-        128.76132098689908
+        5527.366120616061,
```

Comment:
What are these values actually?

Reply:
e.g. throughput or time?

Reply:
cc @rkooo567

Reply:
It's throughput. The second number is standard deviation.